docs/io/visualization/interaction_radius_plot.ipynb

```diff
@@ -0,0 +1,7286 @@
+{
```

Comment:
<p>It's good that you thought about size but we don't really need it because you need to clear outputs before saving - see <a href="https://tardis-sn.github.io/tardis/contributing/development/documentation_guidelines.html#ipynb-documentation" target="_blank">https://tardis-sn.github.io/tardis/contributing/development/documentation_guidelines.html#ipynb-documentation</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>